tools/cli/pkg/command/orchestration.go

```diff
@@ -134,6 +134,40 @@ Operations:
 {{- end }}
 `
 
+var kymaUpgradePreviewTpl = `Kyma Upgrade Preview
+Strategy:         {{.Parameters.Strategy.Type}}
+Schedule:         {{.Parameters.Strategy.Schedule}}
+Workers:          {{.Parameters.Strategy.Parallel.Workers}}
+Kyma Version:     {{with .Parameters.Kyma}}{{.Version}}{{end}}
+Targets:
+{{- range $i, $t := .Parameters.Targets.Include }}
+  {{ orchestrationTarget $t }}
+{{- end -}}
+{{- if gt (len .Parameters.Targets.Exclude) 0 }}
+Exclude Targets:
+{{- range $i, $t := .Parameters.Targets.Exclude }}
+  {{ orchestrationTarget $t }}
+{{- end -}}
+{{- end }}
+`
+
+var kubernetesUpgradePreviewTpl = `Kubernetes Upgrade Preview
+Strategy:         {{.Parameters.Strategy.Type}}
+Schedule:         {{.Parameters.Strategy.Schedule}}
+Workers:          {{.Parameters.Strategy.Parallel.Workers}}
+K8s Version:      {{with .Parameters.Kubernetes}}{{.KubernetesVersion}}{{end}}
```

Comment:
Currently only dynamic KEB determination is supported.

```suggestion
K8s Version:      <determined after start>
```